nonio-ios/Sources/Features/Login/LoginScreen.swift

```diff
@@ -0,0 +1,131 @@
+import SwiftUI
+
+struct LoginScreen: View {
+    @EnvironmentObject var settings: AppSettings
+    @ObservedObject private var viewModel = LoginViewModel()
+    @State private var openURLViewModel = ShowInAppBrowserViewModel()
+
+    var body: some View {
+        NavigationStack {
+            List {
+                R.image.loginAvatar.image
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(maxWidth: .infinity)
+                    .frame(height: 62)
+                    .padding()
+                    .frame(alignment: .center)
+                    .plainListItem()
+                
+                VStack(alignment: .leading) {
+                    Text("CREDENTIALS")
```

Comment:
Generally on the web I'd expect to see this as 
`Text("Credentials").uppercased()` 
or something similar - essentially not encoding the appearance with uppercase characters directly, that way when at some point you move to a string dictionary, it will appear uppercased regardless of what string you insert. Is there a pattern against that in swift, or does it make sense to do it here?

Reply:
Eventually, if we need to support i18n, we'll move it to a dictionary for each language. So uppercase the text is not necessary.